test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -35,12 +31,12 @@ public PackCommandTests(MsbuildIntegrationTestFixture fixture)
         [PlatformFact(Platform.Windows)]
         public void PackCommand_PackNewDefaultProject_NupkgExists()
         {
-            using (var testDirectory = TestDirectory.Create())
+            using (var testDirectory = msbuildFixture.CreateTestDirectory())
             {
                 var projectName = "ClassLibrary1";
                 var workingDirectory = Path.Combine(testDirectory, projectName);
                 // Act
-                msbuildFixture.CreateDotnetNewProject(testDirectory.Path, projectName, " classlib");
+                msbuildFixture.CreateDotnetNewProject(testDirectory.Path, projectName, " classlib -f netstandard2.0");
```

Comment:
Nice change! So no more adapting for future TFM change :)

Reply:
Unfortunately the template doesn't allow arbitrary TFMs. On my machine it only accepts `net5.0`, `netstandard2.1` and `netstandard2.0`. So if a future SDK removes this TFM from the template, our tests will break.

Reply:
I see..Thanks for clarification.